subsys/bluetooth/services/Kconfig.gatts_client

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig BT_GATTS_CLIENT
+	bool "Generic Attribute Profile Service client"
+	select BT_NRF_SERVICES
+	help
+	  Enable Generic Attribute Profile Service client.
```

Comment:
We should have some more descriptions in the help text.
Maybe add:
"The GATT Service Client module can be used to receive service changed indications from a connected peer."

Reply:
Will do.